packages/app/src/components/DBRowSidePanel.tsx

```diff
@@ -189,6 +191,21 @@ export default function DBRowSidePanel({
         ? source.logSourceId
         : undefined;
 
+  const resourceAttributes = useMemo<Record<string, string>>(() => {
+    const key = source.resourceAttributesExpression || 'ResourceAttributes';
+    return normalizedRow?.[key] || {};
+  }, [normalizedRow, source.resourceAttributesExpression]);
+
+  const rumSessionId = useMemo(() => {
+    return (
+      resourceAttributes['rum.sessionId'] ||
+      resourceAttributes['process.tag.rum.sessionId']
+    );
+    // TODO: Add search for related session from logs[?]
+  }, [resourceAttributes]);
```

Comment:
To match what we have in v1, we want to grab the session Id from the top level span. That being said, the session replay should be accessible for any spans in the same trace (log items as well)

<img width="1184" alt="Screenshot 2025-02-24 at 5 11 24 PM" src="https://github.com/user-attachments/assets/50151a68-a8b9-4156-b6a7-b4b8c31cc8a8" />

Reply:
updated